.github/workflows/generate-cve.yml

```diff
@@ -56,9 +63,7 @@ jobs:
         run: |
           cd fleet
           go mod download
-          export VULNCHECK_API_KEY=${{ secrets.VULNCHECK_API_KEY }}
           go run cmd/cve/generate.go --db_dir ./cvefeed --debug
-          rm ./cvefeed/vulncheck.zip
```

Comment:
unused.  cleanup is performed in generate.go

Reply:
something is weird here, that's not in `main`

Reply:
Indeed...

Reply:
Maybe rebase with latest `main` and push force?